v2/README.md

```diff
@@ -0,0 +1,5 @@
+raft-boltdb/v2
+===========
+
+This implementation uses the maintained version of BoltDB, [BBolt](https://github.com/etcd-io/bbolt). This is the primary version of `raft-boltdb` and should be used whenever possible.
```

Comment:
Worth mentioning why it's a v2? I.e. there is no breaking API change to the library it's just the potential for disk format incompatibilities that means we are being conservative and making it a separate import path.

The separate import path is important vs, just a minor version bump because it allows anyone who needs to to import both versions to perform a safe in-place upgrade of old files read with the old version and written back out with the new one.